src/app/Shared/Model/Client_Isolate.model.ts

```diff
@@ -1,13 +1,53 @@
-export type DbCollection = {
-    microorganism: string;
-    samplingContext: string;
-    matrix: string;
-    federalState: string;
-    samplingStage: string;
-    origin: string;
-    category: string;
-    productionType: string;
-}[];
+export type ResistantValues =

```

Comment:
perhaps ResistanceKey?

Reply:
These values aren't used as keys. That's why I don't think renaming is useful.

Reply:
then perhaps ResistantCode? ResistantValues sounds like the values are somehow sticky or permanent. Nevertheless the name should be singular because the type stays for only one of the defined strings.

Reply:
Done. "Rename & Simplify"